### PR TITLE
Properly Track and Change Units on Triangular Test Cut

### PR DESCRIPTION
**Pull request by krkeegan**
The Units button didn't work in the Triangular Test cut page.

### CODE REVIEW

**Comment by blurfl** [comment]
Works as advertised, an important fix. 👍

**Comment by BarbourSmith** [comment]
I think I made the same changes in #497 let me double check to see what I missed

File: UIElements/measureMachinePopup.py [hunk 1]
```diff
@@ -492,11 +492,11 @@ def stopCut(self):
         with self.data.gcode_queue.mutex:
             self.data.gcode_queue.queue.clear()
     
-    def switchUnits(self):
-        if self.unitsBtn.text == 'MM':
-            self.unitsBtn.text = 'Inches'
+    def switchUnits(self, id):
+        if id.text == 'MM':
```
Comment (BarbourSmith) [inline UIElements/measureMachinePopup.py]: This is a really elegant way of doing this! Awesome!

**Comment by BarbourSmith** [comment]
It looks to me like changes with the same effect were made in #497. I'm sorry for leaving it unmerged over the weekend. I figured since the issue had been there since the beginning there was no rush, but it looks like I was wrong 😬 and it was a notable issue this weekend.

Great work tracking down what was going on and fixing it, I am sorry for being slow to merge my PR. I will work on adding @blurfl's suggestion that we have a "sanity check" on values entered right away

**Pull request closed without merging**